x-pack/plugins/beats_management/public/components/table_controls/action_control.tsx

```diff
@@ -60,14 +61,33 @@ export class ActionControl extends React.PureComponent<ActionControlProps, Actio
           <EuiOverlayMask>
             <EuiConfirmModal
               buttonColor={danger ? 'danger' : 'primary'}
-              cancelButtonText="Cancel"
-              confirmButtonText="Confirm"
+              cancelButtonText={
+                <FormattedMessage
+                  id="xpack.beatsManagement.confirmModal.cancelButtonLabel"
```

Comment:
maybe to add .actionControl namespace? => .actionControl.confirmModal

Reply:
I think `.actionControl` is redundant here, because `actionControl` is not an instance.

Reply:
ok